astropy/utils/iers/iers.py

```diff
@@ -667,29 +667,27 @@ def open(cls):
                 return cls.iers_table
 
         dl_success = False
-        err_list = []
+        err_msg = ''
 
-        for url in all_urls:
-            try:
-                filename = download_file(url, cache=True)
-            except Exception as err:
-                err_list.append(str(err))
-            else:
-                dl_success = True
-                break
+        try:
+            filename = download_file(all_urls[0], sources=all_urls, cache=True)
```

Comment:
This now downloads from either URLs but will always cache it under hash of `all_urls[0]`.